forage-android/src/main/java/com/joinforage/forage/android/core/telemetry/Log.kt

```diff
@@ -90,27 +92,27 @@ internal interface Log {
                 }
                 return traceId as String
             }
+
+            override fun addAttribute(key: String, value: Any?): Log {
+                logger?.addAttribute(key, value)
```

Comment:
There is a chance that this function overwrites all attributes on the logger when it's called... I'm not certain but I feel like I vaguely remember that being a reason we only called it once per log. Can you confirm?

Reply:
Based on what I'm seeing here, it seems to cumulatively add context, like Datadog does in JS + iOS

[See logs](https://app.datadoghq.com/logs?query=service:android-sdk%20env:staging%20@payment_method_ref:45de62da41%20&cols=host,service&index=*&messageDisplay=inline&stream_sort=desc&viz=stream&from_ts=1705421316134&to_ts=1705507716134&live=true)